spring/src/main/java/org/axonframework/spring/config/annotation/SpringHandlerEnhancerDefinitionBean.java

```diff
@@ -99,6 +99,7 @@ public void setApplicationContext(ApplicationContext applicationContext) throws
      *
      * @param additionalFactories The extra definitions to register
      * @see SpringBeanParameterResolverFactory
+     * @see SpringBeanDependencyResolverFactory
```

Comment:
Similar as with the `SpringHandlerDefinitionBean`, the `SpringHandlerEnhancerDefinitionBean` is about `HandlerEnhancerDefinition`s, not about `ParameterResolverFactory` instances.
Again, wrongly copied over back in the day, making this added line unnecessary too.